apps/rehabstod/src/components/error/ErrorAlert/ErrorAlert.tsx

```diff
@@ -10,13 +10,15 @@ export function ErrorAlert({
   errorType,
   text,
   dynamicLink,
+  hideErrorId,
   error,
 }: {
   heading: string
   errorType: 'info' | 'attention' | 'success' | 'error'
   text: string
   dynamicLink: boolean
-  error: (FetchBaseQueryError & { id?: string }) | (SerializedError & { id?: string })
+  hideErrorId?: boolean
```

Comment:
Is this used? Could not find an instance where we want to hide error id.

Reply:
Yes for https://inera.atlassian.net/browse/INTYGFV-15725 but I can omit the error instead.